server/pulp/server/db/migrations/0009_qpid_queues.py

```diff
@@ -41,17 +35,17 @@ def migrate(*args, **kwargs):
 
     if not QPID_MESSAGING_AVAILABLE:
         msg = _('Migration 0009 did not run because the python package qpid.messaging is not '
-                'installed. Please install qpid.messaging and rerun the migrations. See %s'
-                'for more information.')
-        msg = msg % QPID_MESSAGING_URL
+                'installed. Pulp\'s Qpid client dependencies can be installed with the '
+                '\"pulp-server-qpid\" package group. See the installation docs for more '
```

Comment:
Since the single quote is being used, the escaping slashes in front of the double quotes aren't needed. They also don't hurt except being slightly less readable when string aren't being interpreted. Not required to change.